src/com/rcv/Tabulator.java

```diff
@@ -329,26 +329,26 @@ private boolean dropUWI(
   // returns: true if any candidates were eliminated
   private boolean dropCandidatesBelowThreshold(
     List<String> eliminated,
-    SortedMap<Float, LinkedList<String>> currentRoundTallyToCandidates
+    SortedMap<BigDecimal, LinkedList<String>> currentRoundTallyToCandidates
   ) {
     if (
-      eliminated.isEmpty() &&
-      config.minimumVoteThreshold() != null &&
-      currentRoundTallyToCandidates.firstKey() < config.minimumVoteThreshold()
+      eliminated.isEmpty() && // <-- TODO: Louis should this condition be here?
```

Comment:
Oh, I'm realizing that when I updated the code to comply with some of the annoying code standards, I didn't do this in the best way. I'm going to change it so that these methods return a list of candidates to eliminate. You can ignore this line or remove it; it doesn't matter right now.